test/cypress/integration/simplified-account/service-settings/email-notifications/email-notifications.cy.js

```diff
@@ -261,4 +264,36 @@ describe('Email notifications settings', () => {
       })
     })
   })
+
+  describe('Email templates', () => {
+    describe('for an admin user', () => {
+      beforeEach(() => {
+        setupStubs()
+        cy.visit(`/simplified/service/${SERVICE_EXTERNAL_ID}/account/test/settings/email-notifications/templates`)
+      })
+      it('should show the Confirmation email tab', () => {
+        cy.get('#tab_confirmation-html').should('have.attr', 'href', '#confirmation-html')
+        cy.get('#add-custom-paragraph-link').should('have.attr', 'href', '#') // TODO verify actual href
+      })
+
+      it('should show the Refund email tab', () => {
+        cy.get('#tab_refund-html').eq(0).should('have.attr', 'href', '#refund-html')
+      })
+    })
```

Comment:
I think it's fine for tests to be slightly less atomic than this, not that it makes much difference here but it could save effort for larger test suites by having multiple content assertions in one test. I won't insist you should change it though, it's fine either way.